redbot/cogs/trivia/data/lists/hockey.yaml

```diff
@@ -0,0 +1,2097 @@
+- Caesar's Palace
+In 1983, the St. Louis Blues were sold to an ownership group that tried to move them to which city? The NHL ultimately rejected this move and found another owner to keep the team in St. Louis. However, as a result of this uncertainty, it caused the Blues to miss the 1983 Entry Draft.:
+- Saskatoon
+The NHL was founded in 1917 by a group of former-NHA owners. The main purpose for founding a new league was to get rid of which owner? He owned the Toronto Blue Shirts.:
+- Eddie Livingstone
+- Livingstone
+Frustrated by the slow drafting process, the Buffalo Sabres drafted an imagery player in the 11th round of the 1974 NHL Amateur Draft. What was the name?:
+- Taro Tsujimoto
+- Tsujimoto
+Cheerleaders and ice girls have long been a part of hockey and other sports. However, in a desperate attempt to sell more seats, which former NHL team hired the girlfriend of a stick boy to streak naked at their games?:
+- California Golden Seals
+Who was both the first Black captain and the first Black coach in the NHL, having served in both capacities for the Chicago Blackhawks?:
+- Dirk Graham
+- Graham
+The first NHL outdoor game occurred in 1954, in Marquette Branch Prison. It was an exhibition game between inmates and which NHL team?:
+- Detroit Red Wings
+- Red Wings
+# Oilers Trivia by A Wild Ferrothorn#0598
+Which team did Wayne Gretzky score his first professional goal against?:
+- The Edmonton Oilers
+- Edmonton Oilers
+- Oilers
+# Predators trivia by -Fulcrum#2658
+Who was the first ever draft pick for the Nashville Predators?:
+- David Legwand
+- Legwand
+Wo was the first Nashville Predator goalie to score a goal?:
+- Chris Mason
+- Mason
+Who was the first goalie to record a shutout for The Vegas Golden Knights?:
+- Oscar Dansk
+- Dansk
+# Ottawa Senators trivia By Cana#3619
+Which coach has the most wins in Ottawa Senators history?:
+- Jacques Martin (341)
+- Jaques Martin
+- Martin
+Who played the most games for the Senators without registering a single point?:
+- Herb Raglan (29 games played, 0 points 1993-94)
+- Herb Raglan
+- Raglan
+Who has the most penalty minutes in Senators history?:
+- Chris Neil (2522)
+- Chris Neil
+- Neil
+Tallest player in Senators history?:
+- Zdeno Chara (6’9’’)
+- Zdeno Chara
+- Chara
+Who is the only player in Senators history to be drafted twice by the team?:
+- G Mathieu Chouinard (1998 1-15, 2000 2-45)
+- Mathieu Chouindar
+- Chouinard
+Who was the first Canadian player drafted by the Senators?:
+- F Chad Penney (1992 2nd)
+- Chad Penney
+- Penney
+First American player drafted by the Senators?:
+- D Jay Kenney (1992 8th)
+- Jay Kenney
+- Kenney
+First Swedish player drafted by the Senators?:
```

Comment:
Plenty of ways to write his name correctly:

Petter Rönnqvist
Petter Rönnquist